### ISSUE TITLE
CORS configuration not working for Tessera

### ISSUE THREAD

**Issue by wxker95** [ConsenSys/quorum-dev-quickstart#153]
I have attempted to enable CORS by adding to the tessera config as shown below.
```
"serverConfigs": [
        {
            "app": "ThirdParty",
            "enabled": true,
            "serverAddress": "http://${HOSTNAME}:9080",
            "communicationType": "REST",
            "cors": {
                "allowedOrigins": ["*"]
            }
        }
        ...
```
However, trying to hit the `/storeraw` endpoint gave the following error:
Access to fetch at 'http://<host>/storeraw' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: It does not have HTTP ok status.

Seems like the preflight request resulted in a 500 status.

Looking in the tessera logs, I see the following lines:
```
{"timestamp":"2022-03-09 08:34:06,094","container":"5385ee3fbd59","level":"INFO","thread":"qtp178371348-25","logger":"o.g.j.s.w.p.WadlModelProcessor$OptionsHandler","message":"Enter Request : OPTIONS : /storeraw","throwable":""}
{"timestamp":"2022-03-09 08:34:06,240","container":"5385ee3fbd59","level":"ERROR","thread":"qtp178371348-25","logger":"c.q.t.a.e.DefaultExceptionMapper","message":"Error occurred: Unable to make protected jakarta.ws.rs.core.Response() accessible: module jakarta.ws.rs does not \"opens jakarta.ws.rs.core\" to module jersey.server. Root cause: Unable to make protected jakarta.ws.rs.core.Response() accessible: module jakarta.ws.rs does not \"opens jakarta.ws.rs.core\" to module jersey.server","throwable":""}
{"timestamp":"2022-03-09 08:34:06,241","container":"5385ee3fbd59","level":"INFO","thread":"qtp178371348-25","logger":"o.g.j.s.w.p.WadlModelProcessor$OptionsHandler","message":"Exit Request : OPTIONS : /storeraw","throwable":""}
{"timestamp":"2022-03-09 08:34:06,241","container":"5385ee3fbd59","level":"INFO","thread":"qtp178371348-25","logger":"o.g.j.s.w.p.WadlModelProcessor$OptionsHandler","message":"Response for storeraw : 500 Internal Server Error","throwable":""}
```

**Comment by joshuafernandes** [comment]
Hi @wxker95 could you confirm the request url you made please? `http:///storeraw` should instead be `http://localhost:9080/storeraw` ? or similar with an IP

**Comment by wxker95** [comment]
Hi @joshuafernandes 

Can confirm that even though the error message shows `http:///storeraw`, it is hitting the correct endpoint. I know this because when using Quorum's 7 node [example](https://github.com/ConsenSys/quorum-examples) with a Tessera version of 0.10.2, I am able to use the storeraw API without issue with the same code. Not sure specifically which Tessera versions have this issue, but it is definitely not working on the main branch of Tessera.

I have tried modifying quorum quickstart's Tessera version to use 0.10.2, but I must not be understanding something about this project because I haven't been able to get it working.

The code I'm using:
```
import Web3 from "web3";
import Web3Quorum from "web3js-quorum";

const web3 = new Web3Quorum(
    new Web3(new Web3.providers.WebsocketProvider(<quorum websocket endpoint>)), 
    { privateUrl: <tessera url>}, 
    true
)
...
//error occurs here
const txHash = await web3.ptm.storeRaw({
      data: <encoded transaction>,
      privateFrom: [...],
});
```

It's worth noting that this is a known issue mentioned on Tessera's github since January: ConsenSys/tessera#1385
I created this issue here before finding it there
